api/tests/unit/organisations/test_unit_organisation_tasks.py

```diff
@@ -0,0 +1,77 @@
+import uuid
+from datetime import timedelta
+
+import pytest
+from django.utils import timezone
+
+from organisations.models import (
+    Organisation,
+    OrganisationRole,
+    UserOrganisation,
+)
+from organisations.tasks import finish_subscription_cancellation
+from users.models import FFAdminUser
+
+
+@pytest.mark.django_db
+def test_finish_subscription_cancellation():
+    # Given
+    organisation1 = Organisation.objects.create()
+    organisation2 = Organisation.objects.create()
+    organisation3 = Organisation.objects.create()
+    organisation4 = Organisation.objects.create()
```

Comment:
The model doesn't doesn't have `blank=False` on it. Maybe you're thinking of a seralizer that enforces it?

Reply:
I was under the impression that `blank=False` was the default (as per [this](https://github.com/django/django/blob/main/django/db/models/fields/__init__.py#L192)). Clearly you're right, I'm just baffled! 😂

Reply:
Huh. Interesting link. I don't know why it works this way for the char field, but I for one like to always specify `blank` and `null` explicitly because when I'm in a rush I don't want to have to look up what the default is for these things.

Reply:
Seems like `CharField` somehow defaults to `''` which I would _still_ expect to fail! Anyway, if I'm being picky, I would rather than these were all instantiated with a name (to avoid this conversation in the future) but I'll approve the PR regardless and you can choose to add the `name` attribute or not.